views/ship-view.es

```diff
@@ -59,11 +58,19 @@ const getTextWidth = text => {
   return Math.ceil(metrics.width)
 }
 
-const ShipName = translate('resources')(({ name, yomi, enemy, t }) => {
-  const translated = t(name)
-  const fullname = ['elite', 'flagship'].includes(yomi)
+const yomis = ['elite', 'flagship']
+
+const getFullname = (t, name, yomi, apiId) => {
+  const translated = t(name, { context: apiId && apiId.toString() })
```

Comment:
How about comparing result with and without context to see if yomi should be appended

Reply:
alright